pkgs/development/python-modules/locationsharinglib/default.nix

```
@@ -0,0 +1,65 @@
+{ lib
+, betamax
+, buildPythonPackage
+, cachetools
+, coloredlogs
+, emoji
+, fetchFromGitHub
+, nose
+, python
+, pythonOlder
+, pytz
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "locationsharinglib";
+  version = "4.1.6";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "costastf";
+    repo = pname;
+    rev = version;
+    sha256 = "1f1wgxg7csl5aqd2c50grirps7sjry1qi47c8f2z31l2ssyxpm10";
+  };
+
+  propagatedBuildInputs = [
+    coloredlogs
+    requests
+    cachetools
+    pytz
+  ];
+
+  checkInputs = [
+    betamax
+    emoji
+    nose
+  ];
+
+  postPatch = ''
+    # Requires a .VERSION file
+    substituteInPlace locationsharinglib/__init__.py \
+      --replace "from ._version import __version__" "" \
+      --replace "assert __version__" ""
```

could we also just do
```suggestion
    echo ${version} > .VERSION
```

---

I switch to the source from PyPI. There the `.VERSION` is present in the right location.